### DISCUSSION TITLE
sodium error when launching

### DISCUSSION THREAD

**Discussion by jlockman2** [Intermesh/groupoffice#754] [Q&A] [answered]
I have installed Group Office on our server and when I launch the URL I get the following error:  ErrorException in Unknown at line 0: Module 'sodium' already loaded at 2021-12-08T20:57:47+01:00  --  I previously installed sodium PHP extension as required for another application.  Is there a way to resolve this for Group Office?

**Comment by jlockman2** [comment]
This is resolved - I changed the PHP version for this subdomain (sodium not installed on this version) to a different version than the PHP version for the domain for other application (sodium installed for this one) and uninstalled/reinstalled.

**Comment by mschering** [comment, marked as answer]
Great, the error arises when the module sodium.so is loaded twice in the php configuration.
